### DISCUSSION TITLE
"show_missing" is kind of ignored

### DISCUSSION THREAD

**Discussion by Riddler84** [meisnate12/Plex-Meta-Manager#348] [Q&A] [answered]
Hi, is there any way to completely disable the processing of missing movies? 
I have set `show missing` and `save_missing` to false, but it still processes them in the background after each collection is built and then prints out a number of how many movies from that imdb list (for example) are missing. 
This takes a lot of time, especially if the list is big. And I don't really care about this info, so is there any way to skip this? Thanks

**Comment by meisnate12** [comment, marked as answer]
just added in develop that it will only process missing shows if theres a point

**Reply by Riddler84** [reply]
Thx!
